src/Components/CategoriesContainer.tsx

```diff
@@ -7,176 +7,213 @@ import { useNavigate } from "react-router-dom";
 import "aos/dist/aos.css";
 
 const CategoryCard = () => {
-  const navigate = useNavigate();
-  AOS.init({ once: true });
-  const { categories, fetchCategoriesFromDb, setSelectedCategoryId } =
-    useItems();
-  const [showMore, setShowMore] = useState<boolean>(false);
+    const navigate = useNavigate();
+    AOS.init({ once: true });
+    const { categories, fetchCategoriesFromDb, setSelectedCategoryId } =
+        useItems();
+    const [showMore, setShowMore] = useState<boolean>(false);
 
-  const slicedCategories = showMore ? categories : categories.slice(0, 4);
+    const slicedCategories = showMore ? categories : categories.slice(0, 4);
 
-  useEffect(() => {
-    fetchCategoriesFromDb();
-  }, []);
+    useEffect(() => {
+        fetchCategoriesFromDb();
+    }, []);
 
-  const test = (id: string) => {
-    setSelectedCategoryId(id);
-  };
-  return (
-    <Box sx={categoriesContainer}>
-      <Box>
-        <Typography sx={secTitle}>Explore our categories</Typography>
-      </Box>
-      <Box sx={categoriesDiv}>
-        {slicedCategories?.map((category: any) => (
-          <Box
-            sx={categoryDiv}
-            key={category.id}
-            onClick={() => {
-              navigate(`/category/${category.title}`);
-              test(category.id);
-            }}
-            data-aos="fade-left"
-            data-aos-offset="200"
-            data-aos-duration="1000"
-          >
-            <Box sx={categoryTitleDiv}>
-              <Typography sx={categoryTitle}>{category.title}</Typography>
+    const test = (id: string) => {
```

Comment:
Please rename 'test' to something more explanatory :)

Reply:
Whaaat I did not write that 😂 will change it!